libs/designsystem/badge/src/badge.component.scss

```diff
@@ -20,7 +20,8 @@
     box-shadow: var(--kirby-badge-elevation, none);
   }
 
-  ::slotted(kirby-icon) {
+  /* stylelint-disable selector-pseudo-element-no-unknown */
+  ::ng-deep kirby-icon {
```

Comment:
Please use helper method instead to align with rest of code base—and for readability and intent:
```suggestion
  @include utils.slotted(kirby-icon) {
```

Reply:
Nice, fixed in 26154290a 👍